tests/unit/test_composition_manifest_consistency.py

```
@@ -0,0 +1,60 @@
+"""composition と manifest の整合性テスト（ADR-015 準拠）。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)